pootle/apps/pootle_fs/management/commands/__init__.py

```diff
@@ -0,0 +1,128 @@
+            raise CommandError(e)
+
+    @lru_cache()
+    def get_project(self, project_code):
+        try:
+            return Project.objects.get(code=project_code)
+        except Project.DoesNotExist as e:
+            raise CommandError(e)
+
+
+class FSAPISubCommand(ProjectSubCommand):
+
+    def add_arguments(self, parser):
+        super(FSAPISubCommand, self).add_arguments(parser)
+        parser.add_argument(
+            '-p', '--fs_path',
+            action='store',
+            dest='fs_path',
+            help='Filter translations by filesystem path')
+        parser.add_argument(
+            '-P', '--pootle_path',
+            action='store',
+            dest='pootle_path',
+            help='Filter translations by Pootle path')
+
+    def handle_api_options(self, options):
+        return {
+            k: v
+            for k, v
+            in options.items()
+            if k in [
+                "pootle_path", "fs_path",
+                "force", "pootle_wins"]}
+
+    def handle_api(self, **options):
+        api_method = getattr(
+            self.get_fs(options["project"]),
+            self.api_method)
+        return api_method(**self.handle_api_options(options))
+
+    def display(self, **options):
+        return ResponseDisplay(
+            self.handle_api(**options))
+
+    @property
+    def colormap(self):
+        return RESPONSE_COLORMAP
+
+    @lru_cache()
+    def get_style(self, key):
+        pootle_style, fs_style = self.colormap.get(key, (None, None))
+        if pootle_style:
+            pootle_style = getattr(self.style, "FS_%s" % pootle_style)
+        if fs_style:
+            fs_style = getattr(self.style, "FS_%s" % fs_style)
+        return pootle_style, fs_style
+
+    def is_empty(self, display):
+        return not display.context.made_changes
+
+    def handle(self, *args, **options):
+        display = self.display(**options)
+        if self.is_empty(display):
+            self.stdout.write(str(display).strip())
+            self.stdout.write("")
+            return
+        for section in display.sections:
+            section = display.section(section)
+            self.stdout.write(section.title, self.style.HTTP_INFO)
+            self.stdout.write("-" * len(section.title), self.style.HTTP_INFO)
+            if section.description:
+                self.stdout.write(section.description)
+            self.stdout.write("")
+            for item in section.items:
+                pootle_style, fs_style = self.get_style(item.section)
+                self.write_line(
+                    item.pootle_path,
+                    item.fs_path,
+                    pootle_style=pootle_style, fs_style=fs_style)
+            self.stdout.write("")
+        self.stdout.write("")
+
+    def write_line(self, pootle_path, fs_path,
+                   pootle_style=None, fs_style=None):
+        self.stdout.write("  %s" % pootle_path, pootle_style)
```

Comment:
@unho you are - what is here is correct - you need to run this if you want to see the output